doc/fluid/design/data_pipeline/data_pipeline.md

```diff
@@ -0,0 +1,92 @@
+# Design Doc: Fluid Data Pipeline
+
+This document is about how Fluid training and inference programs reads data.
+
+## Standard Data Format
+
+### Case 1: Data from Files
+
+Consider a Fluid trianing program, `resnet50.py`, needs to read data from disk:
+
+```bash
+cat data | python resnet50.py
+```
+
+The fact that the who collect the data might not be the same person who wrote `resnet50.py` inspires that
+
+1. Fluid operators used in `resnet50.py` need to recognize the file format of `data`, or, we need a standard data format.
+1. These operators need to be able to read the standard input.
```

Comment:
read the standard input -> read **from** the standard input

Reply:
Done